### ISSUE TITLE
version not quite right in netCDF attributes

### ISSUE THREAD

**Issue by taobrienlbl** [LBL-EESA/TECA#746]
running `ncdump -h` on a TECA-produced netCDF file shows version information that doesn't seem quite right:

```
// global attributes:
		:TECA_version = "4.0.0-572-gf2ef981" ;
```

The two things that stand out to me are: (1) TECA's current release version 5.0.0, and (2) the commit hash seems to have an extra 'g' in front. This application was built from commit f2ef981, and I see that that commit hash is part of the version number--but there looks to be an extra 'g' in front of it.  I'm not sure whether that's intentional.

**Comment by taobrienlbl** [comment]
okay, ignore the part about the `g` -- I see that's output from `git describe`. But it looks like `git describe` is finding 4.0.0 as the release and not 5.0.0.  This might be because 4.0.0 is the last annotated tag?

**Comment by burlen** [comment]
yes the number here 572 tells how many commits past the tag, g tells the specific sha you're at. the problem is that the info is cached, cmake will not update it unless you wipe the cache. We can fix this by forcing it to update every time

**Comment by burlen** [comment]
as an iterim measure you can open up the CMakeCache.txt file and delete the TECA_VERSION_DESCR line and when you make the version will be updated.

**Comment by burlen** [comment]
fixed by https://github.com/LBL-EESA/TECA/pull/742/commits/4849017b5934cd793e4c4c3d5b584c09dd884ab9
